content/governing-content/content-lifecycle/signing-off-content/sign-off-template/template.md

```diff
@@ -12,10 +12,11 @@ We’re asking for your sign off to publish:
 [insert staging URL if appropriate].
 
 Below is a checklist of what we’ve already validated.
+
 We’d specifically like you to review this content for the following:
 
-organisational risk
-specific concerns such as [insert as appropriate]
+• organisational risk
```

Comment:
are these the correct bullet @joolswood

Reply:
Can't seem to get markdown versions to show up?

Reply:
did you try with a space between the - and text?

Reply:
Yes

Reply:
They just render as a line break